src/app/base/components/FormikFormButtons/_index.scss

```diff
@@ -43,5 +43,18 @@
         width: 100%;
       }
     }
+
+    @media only screen and (max-width: $breakpoint-large) {
+      .formik-form-buttons__container {
+        display: flex;
+        flex-direction: column-reverse;
```

Comment:
I appreciate your initiative here, but having thought about this a bit more I'm not sure this is needed.

It's not clear if the reverse button order on mobile by default is best from UX perspective. But also this has wide implications - you'd need to check it's appropriate and works well for every case where these buttons are used.

Reply:
Okay, I'll make the change now